hydra-node/test/Test/Util.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
Why is this pragma needed?

Reply:
Because of the added `trace` call in the `traceDebug` function.